lookout/style/typos/__main__.py

```diff
@@ -0,0 +1,46 @@
+import argparse
+import sys
+
+from lookout.core.cmdline import ArgumentDefaultsHelpFormatterNoNone
+from lookout.style.typos.nn_prediction import create_and_train_nn_prediction_from_file
+from lookout.style.typos.utils import CORRECT_TOKEN_COLUMN, TYPO_COLUMN
+
+
+def create_parser():
+    parser = argparse.ArgumentParser(formatter_class=ArgumentDefaultsHelpFormatterNoNone)
+    subparsers = parser.add_subparsers(help="Commands", dest="command")
```

Comment:
no need for subparsers if you have only one command (it's the case here). You can remove this line, the add_parser function, the `vis_parser` declaration and use `parser` instead of `vis_parser` :)

You can also remove the `set_defaults(handler=...` line, it's a trick to handle multi-commands properly but there is only one command here.